Common subdirectories: zebu-3.5.5.orig/doc and zebu-3.5.5/doc
Common subdirectories: zebu-3.5.5.orig/test and zebu-3.5.5/test
Only in zebu-3.5.5: zebra-debug.fasl
Only in zebu-3.5.5: zebu-actions.fasl
Only in zebu-3.5.5: zebu-asdf-setup.fasl
Only in zebu-3.5.5: zebu-aux.fasl
diff -u zebu-3.5.5.orig/zebu-aux.lisp zebu-3.5.5/zebu-aux.lisp
--- zebu-3.5.5.orig/zebu-aux.lisp	1998-12-09 14:22:24.000000000 -0600
+++ zebu-3.5.5/zebu-aux.lisp	2004-11-07 17:15:37.400965600 -0600
@@ -244,3 +244,9 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;                           End of zebu-aux.lisp
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+;; http://www.sbcl.org/manual/Defining-Constants.html#Defining%20Constants
+
+(defmacro define-constant (name value &optional doc)
+  `(defconstant ,name (if (boundp ',name) (symbol-value ',name) ,value)
+     ,@(when doc (list doc))))
\ No newline at end of file
Only in zebu-3.5.5: zebu-aux.lisp~
Only in zebu-3.5.5: zebu-closure.fasl
Only in zebu-3.5.5: zebu-compile-mg.fasl
Only in zebu-3.5.5: zebu-compile.fasl
Only in zebu-3.5.5: zebu-driver.fasl
Only in zebu-3.5.5: zebu-dump.fasl
Only in zebu-3.5.5: zebu-empty-st.fasl
Only in zebu-3.5.5: zebu-first.fasl
Only in zebu-3.5.5: zebu-follow.fasl
Only in zebu-3.5.5: zebu-g-symbol.fasl
Only in zebu-3.5.5: zebu-generator.fasl
diff -u zebu-3.5.5.orig/zebu-generator.lisp zebu-3.5.5/zebu-generator.lisp
--- zebu-3.5.5.orig/zebu-generator.lisp	1999-01-13 12:16:30.000000000 -0600
+++ zebu-3.5.5/zebu-generator.lisp	2004-11-07 17:20:28.192758528 -0600
@@ -421,7 +421,7 @@
 ; return: (1) ((<test for print-case> <format stmt derived from syntax>) ..)
 ;         (2) a lambda-list binding the %u .. variables used to accessors
 ;             derived from the paths.
-(defconstant *vars-to-use* '("%R" "%S" "%T" "%U" "%V" "%W" "%X" "%Y" "%Z"))
+(define-constant *vars-to-use* '("%R" "%S" "%T" "%U" "%V" "%W" "%X" "%Y" "%Z"))
 
 (defun gen-clauses (clauses KB-sequence-print-fn-AL
 			    &aux (vars-to-use (mapcar #'intern *vars-to-use*))
Only in zebu-3.5.5: zebu-generator.lisp~
Only in zebu-3.5.5: zebu-kb-domain.fasl
Only in zebu-3.5.5: zebu-lalr1.fasl
Only in zebu-3.5.5: zebu-loader.fasl
Only in zebu-3.5.5: zebu-loadgram.fasl
Only in zebu-3.5.5: zebu-lr0-sets.fasl
Only in zebu-3.5.5: zebu-mg-hierarchy.fasl
diff -u zebu-3.5.5.orig/zebu-mg.tab zebu-3.5.5/zebu-mg.tab
--- zebu-3.5.5.orig/zebu-mg.tab	2002-05-24 14:01:41.000000000 -0500
+++ zebu-3.5.5/zebu-mg.tab	2004-11-07 17:15:49.062192824 -0600
@@ -1,5 +1,5 @@
 
-(:FILE "/home/rudi/lisp/zebu-3.5.5/zebu-mg.zb" :NAME "zebu-mg" :DOMAIN-FILE "zmg-dom" :PACKAGE "ZEBU" :GRAMMAR "null-grammar" :IDENTIFIER-START-CHARS "$-_abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ" :IDENTIFIER-CONTINUE-CHARS "$-_.abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890" :DOMAIN-FILE "/home/rudi/lisp/zebu-3.5.5/zmg-dom.lisp")
+(:FILE "/mnt/space/tmp/zebu-3.5.5/zebu-mg.zb" :NAME "zebu-mg" :DOMAIN-FILE "zmg-dom" :PACKAGE "ZEBU" :GRAMMAR "null-grammar" :IDENTIFIER-START-CHARS "$-_abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ" :IDENTIFIER-CONTINUE-CHARS "$-_.abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890" :DOMAIN-FILE "/mnt/space/tmp/zebu-3.5.5/zmg-dom.lisp")
 #37(THE-EMPTY-STRING AUGMENTED-START THE-END-G-SYMBOL META-GRAMMAR DEF-TYPE ZB-RULE IDENTIFIER ":=" TYPED-CONJ PRINT-FUNCTION ";" CONJ "<<" "print-function:" ">>" ":" "[" LABEL-VALUE-PAIRS "]" FEAT-TERM NUMBER STRING "*" "+" LABEL-VALUE-PAIR "(" ")" NON-TERMINAL "-->" RHS RHS1 MORE-RHS CONSTITUENT-LIST "{" "}" CONSTITUENT "|" )
 
 
Only in zebu-3.5.5: zebu-oset.fasl
Only in zebu-3.5.5: zebu-package.fasl
Only in zebu-3.5.5: zebu-printers.fasl
Only in zebu-3.5.5: zebu-regex.fasl
Only in zebu-3.5.5: zebu-slr.fasl
Only in zebu-3.5.5: zebu-tables.fasl
Only in zebu-3.5.5: zebu-tree-attributes.fasl
Only in zebu-3.5.5: zmg-dom.fasl
Only in zebu-3.5.5: zmg-dom.lisp
